utils/frame/benchmarking-cli/src/pallet/command.rs

```diff
@@ -422,7 +426,7 @@ impl PalletCmd {
 			if let Some(path) = &self.json_file {
 				fs::write(path, json)?;
 			} else {
-				println!("{}", json);
+				log::info!(target:LOG_TARGET,"{}", json);
```

Comment:
```suggestion
				print!("{json}");
```

Reply:
My comment from above was that mixing `println` and `log` messes with the ordering since they have different output buffers.  
I am not blocking this MR for that, but just something to consider.

Reply:
Yeah for sure, I know. However the idea for this stuff is always that you log to stderr (what we do) and the actual data you produce is printed to stdout (what we also do here). Then you can just pipe the output to some file `executable something > output`. Then you only redirect the output and still see the log lines.

Reply:
Ah okay, makes sense.